modules/custom-status/custom-status.php

```diff
@@ -87,10 +91,9 @@ public function init() {
 		// Pagination for custom post statuses when previewing posts
 		add_filter( 'wp_link_pages_link', [ $this, 'modify_preview_link_pagination_url' ], 10, 2 );
 
-		// REST endpoints
-		CustomStatusEndpoint::init();
-
 		add_filter( 'user_has_cap', [ $this, 'remove_or_add_publish_capability_for_user' ], 10, 3 );
+
+		add_filter( 'vw_custom_status_ui_additions', [ $this, 'add_custom_status_ui_properties' ] );
```

Comment:
Statuses are stored with `required_user_ids`, but this isn't enough to display corresponding `user_login` string in the UI. This filter is used for custom status preprocessing on initial page load as well as REST responses to fill in `required_user_logins` to supply a value that can be displayed in the UI.